src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,12 +1,30 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
+import java.time.Instant;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+
+    public static final DateTimeFormatter DATE_FORMATTER_ENGLISH
+            = DateTimeFormatter.ofPattern("d MMM yyyy", Locale.ENGLISH);
+    public static final DateTimeFormatter DATE_FORMATTER_YYYY_MM_DD
+            = DateTimeFormatter.ofPattern("yyyy MM dd");
+    public static final DateTimeFormatter FULL_DATE_TIME_FORMATTER_ENGLISH
+            = DateTimeFormatter.ofPattern("dd MMMM yyyy HH:mm", Locale.ENGLISH);
+    public static final ZoneOffset UKRAINE_ZONE_OFFSET = ZoneOffset.of("+02:00");
+    public LocalDate localDateNow = LocalDate.now();
```

Comment:
let's move this field to all methods where it is needed

Reply:
Ok, will do. Could you please explain what is the advantage of doing so?